src/platform/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -275,6 +264,35 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
         return undefined;
     }
 
+    // Use our jupyter session to delete all the cells
+    private async deleteDumpCells() {
+        const fileValues = [...this.cellToFile.values()];
+        // Need to have our Jupyter Session and some dumpCell files to delete
+        if (this.jupyterSession && fileValues.length) {
+            // Create our python string of file names
+            const fileListString = fileValues
+                .map((filePath) => {
+                    return '"' + filePath + '"';
+                })
+                .join(',');
+
+            // Insert into our delete snippet
+            const deleteFilesCode = `import os
+_VSCODE_fileList = [${fileListString}]
```

Comment:
This bit works I believe since the file names are coming from jupyter creating them. So I'm not worried about having a " or something like that in the file string.

Reply:
Is there a way to log errors just in case they ever happen?

Reply:
Disregard! I just read the `traceErrors: true`

Reply:
Yeah, the trace should at least put something into our logging (but nothing user visible).

Reply:
Thanks!